### DISCUSSION TITLE
Finetune Language model

### DISCUSSION THREAD

**Discussion by wuchaowei2012** [PaddlePaddle/PaddleSpeech#588] [Ideas]
使用 预留下载的 语言模型 Language （Mandarin LM Small）， 发现对 新词的支持不够， 如“新冠肺炎”， 希望能 对 Mandarin LM Small 这些模型 进行 finetune， 不知道 如何开始做。

**Comment by zh794390558** [comment]
这种新词可以用hotword的方式解决。具体的tinetune的方法可以参看kenlm的文档，具体没有操作过。

**Reply by wuchaowei2012** [reply]
hotword 怎么实现啊？ 同学，可以 给一个 url么？

**Reply by zh794390558** [reply]
现在repo里没有加这块儿的东西，之后会加，暂时没有时间。原理和LM或这Dict是一样的，额外给hotword一个分数增益。
